Add new functions

ravanelli opened the pull request:
- run_cmd
 - tag_image_and_return_list

Signed-off-by: Renata Ravanelli <rravanel@redhat.com>

miabbott commented on main.py:
```diff
@@ -187,6 +206,24 @@ def delete_image(region_id, image_id, check_tag_key=None, check_tag_value=None):
 # Takes a release version (i.e. 4.10) as an argument
 #
 # Returns oa dict keyed off of build ID with values like {region_id: {release: build_id, image: image_id}}
+def run_cmd(command, silent = False, ignore_error = False, dry_run=True):
```
Can we feed functions into this?

I think we can just simplify this to just adjusting any functions that are doing "write" operations (i.e. delete images, tagging, etc) would check for the presence of a dry-run arg and only print what they would do, rather than execute it.

ravanelli replied:
@miabbott  That's the idea, call all delete, tag, write stuff via run_cmd, with the default as dry run.

miabbott commented on main.py:
```diff
@@ -133,6 +134,25 @@ def tag_image(region_id, image_id, tag_key=None, tag_value=None):
+# tag value
+#
+# Returns a JSON file path
+def tag_image_and_return_list(image_list, tag_key=None, tag_value=None):
+    images = []
+    tmpdir = tempfile.mkdtemp()
+    file_path = os.path.join(tmpdir, 'tagged_images.json')
```
We might want to have a `filename` argument here where we can pass in an existing/well-known location

That way if we end up re-running the script, we can pick up where we left off.

ravanelli committed:
Add new functions

 - run_cmd: Run the command or dry run it
 - tag_image_and_return_list: Return a json file with the images tagged
